content/blog/2018/12/2018-12-22-gsoc-2019-call-for-mentors.adoc

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "Google Summer of Code 2019 Call for Mentors"
+tags:
+- gsoc
+- gsoc2019
+- community
+- events
+author: martinda
+links:
+  gitter: /jenkinsci/gsoc-sig
+  googlegroup: jenkins-gsoc-all-public
+  sig: gsoc
+---
+
+NOTE: link:https://developers.google.com/open-source/gsoc/[Google Summer of Code]
+is as program where students are paid a stipend by Google to work on a free open source project like Jenkins,
+at full-time for four months (May to August).
+Mentors get actively involved with students starting at the end of February when students start to apply
+(see the link:https://developers.google.com/open-source/gsoc/timeline[timeline]).
+
+image:/images/gsoc/jenkins-gsoc-logo_small.png[Jenkins GSoC, role=center, float=right]
+
+We are looking for mentors and project ideas to participate in the 15th edition of the Google Summer of Code program!
+We have until February 6th, 2019 at 8pm UTC to submit the application on behalf the Jenkins Organization, but obviously,
+we want to be ready before that.
+
+The first step in the process is to have mentors and project ideas.
+Then we will apply to Google.
+We need Google to accept Jenkins' application to the program itself.
+And for this to happen, *we need project proposals and mentors*.
+
+We currently have a link:/projects/gsoc/2019/project-ideas[list of project idea proposals],
+and are still accepting new project proposals, mentors, technical advisers, and subject matter experts.
+
+Making a project idea proposal is easy, link:/projects/gsoc/proposing-project-ideas[read the quick start instructions]
```

Comment:
Maybe makes sense to just copy-paste steps and put them here so that it takes less time